docs/environment_setup.rst

```diff
@@ -0,0 +1,72 @@
+.. _environment_setup:
+
+Requirements and setup
+######################
+
+This page outlines the requirements that you need to meet before you start working with the KNX IoT Point API environment.
+Once completed, you will be able to run and test a sample KNX IoT application.
+
+Hardware requirements
+*********************
+
+To meet the hardware requirements, ensure you have two `nRF52840 DK`_ boards.
+To start working with the devices, refer to the `Getting started with nRF52 Series`_ page.
+
+.. _sniffer_configuration:
+
+Optionally, you can use `nRF52840 Dongle`_ as a Thread sniffer.
+To start working with the `nRF52840 Dongle`_ as a sniffer, refer to the `nRF Sniffer for 802.15.4`_ page.
+
+Software requirements
+*********************
+
+To meet the software requirements, install `nRF Connect SDK v2.3.0`_ and `nRF Command Line Tools`_.
+
+nRF Connect SDK
+===============
+
+Perform the following steps to install `nRF Connect SDK v2.3.0`_ and setup `nRF52840 DK`_ boards:
+
+#. Set up your development environment by choosing one of the installation methods below:
+
+   * Follow `Installing automatically`_ guildelines to perform an automatic installation through the Toolchain Manager.
+   * Follow `Installing manually`_ guidelines to perform a manual installation.
+
+   .. note::
+      For additional information on setting up the device as well as Nordic’s development environment and tools, see the `nRF Connect SDK Getting started guide`_.
+
+#. Clone the KNX IoT Point API repository.
+
+   .. code-block:: console
+
+      $ git clone https://github.com/NordicPlayground/nrf-knx-iot.git
+
+#. Download dependencies.
+
+   .. code-block:: console
+
+      $ git submodule init
+      $ git submodule update
+
+nRF Command Line Tools
+======================
+
+Download the nRF Command Line from the `nRF Command Line Tools`_ page.
+
+Wireshark
+=========
+
+Optionally, for tracing and sniffing purposes, install `Wireshark`_.
+To configure Wireshark for Thread packets sniffing, go to the `Configuring Wireshark for Thread`_ page.
```

Comment:
Infocenter is a private website. Should we link to any other public pages, or add guidelines directly here?